### DISCUSSION TITLE
I want to build security onion and test it and install it in client environment

### DISCUSSION THREAD

**Discussion by sudesh0sudesh** [Security-Onion-Solutions/securityonion#12011] [Unsupported Versions]
How can i do it without making multiple configuration changes and issues with IPs, I know i can use DHCP but that would prevent me from gaining access to the system if i am not logging in from whitelisted subnet

**Comment by RagnarSTS** [comment]
SIEMs pretty much always need a lot of care and feeding at the beginning.  You will always need to do deep dives into configuration changes during and after deployment.
One will need to write down the configuration and setup requirements made during testing, and be sure to create a cut sheet of exactly how to duplicate the tasks with the new production IPs.  
DHCP is never the answer for a server.

**Comment by TotieBash** [comment]
The way I like to do it on my POC is to install using "Standalone" mode and then "so-ip-update" to change/update IP address.
